doc/api/fs.markdown

```diff
@@ -577,7 +577,11 @@ If you want to be notified when the file was modified, not just accessed
 you need to compare `curr.mtime` and `prev.mtime`.
 
 _Note: when an `fs.watchFile` operation results in an `ENOENT` error, it will
-invoke the callback once. This is a change in functionality since v0.10._
+ invoke the listener once, with two stat objects. All the fields in those stat
```

Comment:
Since `two stat objects` is redundant - it always does two stat objects - you could change it to something like `invoke the listener once, with all fields zeroed (or, for dates, the Unix Epoch).`